model/storage/chain/constants.py

```diff
@@ -1,2 +0,0 @@
-# The uid for this subnet.
-SUBNET_UID = 9
```

Comment:
Does the chain metadata store not need to be updated to get the subnet uid from pretrain now?

In retrospect maybe this should just be at a higher level in the repo?

Reply:
Already done